nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/state/system-diagnostics/index.ts

```diff
@@ -91,6 +91,7 @@ export interface SystemDiagnosticSnapshot {
 
 export interface SystemDiagnosticsRequest {
     nodewise: boolean;
+    errorStrategy: 'banner' | 'snackbar';
```

Comment:
I don't think the `errorStrategy` is needed when opening this from `navigation`. Can we either have a different action payload for `reload` (where `errorStrategy` is needed) or have it be optional here.